docs/AboutUs.md

```diff
@@ -9,15 +9,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Zechen Lin
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/bubbl3t.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/Bubbl3T)]
+[[portfolio](team/bubbl3t.md)]
 
-* Role: Project Advisor
+* Role: Team Lead + IntelliJ Expert
+* Responsibilities: Deliverables and deadlines, Integration, Logic, Model
```

Comment:
Add full stops in the end.

Reply:
We should do a standardization closer to the deadline, I have created a new issue for it.